samples/graalvm-native-jna/gradle/libs.versions.toml

```diff
@@ -0,0 +1,34 @@
+[versions]
+jna = "5.15.0-SNAPSHOT"
```

Comment:
note for release: should eventually be pinned to minimum support version for gvm